### ISSUE TITLE
Error using process()

### ISSUE THREAD

**Issue by arthaud** [binjitsu/binjitsu#67]
I'm getting an error when I try to use pwnlib.tubes.process:

```python
from pwn import *
p = process(which('python'))
```

The traceback:

```
Traceback (most recent call last):
  File "/tmp/test.py", line 2, in <module>
    p = process(which('python3'))
  File "/tmp/env/lib/python2.7/site-packages/binjitsu-2.2.1-py2.7.egg/pwnlib/tubes/process.py", line 284, in __init__
    preexec_fn = self.__preexec_fn)
  File "/usr/lib64/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/usr/lib64/python2.7/subprocess.py", line 1335, in _execute_child
    raise child_exception
IOError: [Errno 3] No such process
```

I found out that is due to:
```python
with open('/proc/self/coredump_filter', 'w') as f:
    f.write('0xff')
```
in pwnlib/tubes/process.py

I'm getting a `ProcessLookupError: [Errno 3] No such process` when I try to run it in a python interactive interpreter.

Python 2.7.11
Upstream version of binjitsu
Archlinux
Linux 4.3.3-3-ARCH #1 SMP PREEMPT Wed Jan 20 08:12:23 CET 2016 x86_64 GNU/Linux

**Comment by zachriggle** [comment]
`ProcessLookupError` is a Python3 exception

**Comment by zachriggle** [comment]
Fixed in 541734a
